### DISCUSSION TITLE
"Panthor" DRM Driver Coming Together For Newer Arm Mali GPUs

### DISCUSSION THREAD

**Discussion by ferrellsl** [Joshua-Riek/ubuntu-rockchip#500] [General]
Just thought I'd share this as many RK3588 users  are looking forward to Vulkan support.

https://www.phoronix.com/news/Panthor-DRM-Newer-Mali

**Comment by Joshua-Riek** [comment]
I've already done some testing with this. However, X11 does not work, and I don't know enough about Mesa or DRM driver internals. But its good progress.

**Comment by bagong** [comment]
DRM means "direct rendering manager" here. Wikipedia:

"The Direct Rendering Manager (DRM) is a subsystem of the Linux kernel
responsible for interfacing with GPUs of modern video cards."
